### ISSUE TITLE
Docs: Español Setup Guide Support

### ISSUE THREAD

**Issue by BootsinSoots** [ArchipelagoMW/Archipelago#2301]
### What feature would you like to see?

In light of the recent conversations regarding individuals interested in assisting with guides in other languages, I am compiling a list of setup guides that do not have a guide in the languages for which I am aware of us have volunteers for. I realize this may never be a fully complete issue, but I felt it was a good idea to have this as a recorded issue to strive for.

Currently, I am aware of 4 languages beyond English that have volunteers: Deutsche (German), Español (Spanish), Français (French), and Polski (Polish). I will organize the missing documentation by language, into separate issues because _oh boy was this list longer than I expected._

- [ ] Archipelago: Getting Started
- [ ] Archipelago: Archipelago Setup Guide for Mac
- [ ] Archipelago: Archipelago Server and Client Commands
- [ ] Archipelago: Advanced YAML Guide
- [ ] Archipelago: Archipelago Triggers Guide
- [ ] Archipelago: Archipelago Plando Guide
- [ ] Adventure: Setup guide
- [ ] Blasphemous: Setup Guide
- [ ] Bumper Stickers: Setup Guide
- [ ] Checksfinder: Setup Guide
- [ ] Clique: Setup Guide
- [ ] Dark Souls III: Setup Guide
- [ ] DLCQuest: Setup Guide
- [ ] Donkey Kong Country 3: Setup Guide
- [ ] Doom 1993: Setup Guide
- [ ] Factorio: Setup Guide
- [ ] Final Fantasy: Setup Guide
- [ ] Hollow Knight: Setup Guide
- [ ] Hylics 2: Setup Guide
- [ ] Kingdom Hearts 2: Setup Guide
- [ ] The Legend of Zelda: Setup Guide
- [ ] A Link to the Past: Plando Tutorial
- [ ] A Link to the Past: 'OOF' Sound replacement
- [ ] Links Awakening DX: Setup Guide
- [ ] Lufia II Ancient Cave: Setup Guide
- [ ] MegaMan Battle Network 3: Setup Guide
- [ ] Meritous: Setup Guide
- [ ] The Messenger: Setup Guide
- [ ] Muse Dash: Setup Guide
- [ ] Noita: Setup Guide
- [ ] Overcooked! 2: Setup Guide
- [ ] Raft: Setup Guide
- [ ] Risk of Rain 2: Setup Guide
- [ ] Rogue Legacy: Setup Guide
- [ ] Secret of Evermore: Setup Guide
- [ ] Slay the Spire: Setup Guide
- [ ] SMZ3: Setup Guide
- [ ] Sonic Adventure 2 Battle: Setup Guide
- [ ] Starcraft 2 Wings of Liberty: Setup Guide
- [ ] Stardew Valley: Setup Guide
- [ ] Subnautica: Setup Guide
- [ ] Sudoku: Setup Guide
- [ ] Super Mario 64: Setup Guide
- [ ] Super Mario World: Setup Guide
- [ ] Super Metroid: Setup Guide
- [ ] Terraria: Setup Guide
- [ ] Timespinner: Setup Guide
- [ ] Undertale: Setup Guide
- [ ] VVVVVV: Setup Guide
- [ ] Wargroove: Setup Guide
- [ ] The Witness: Setup Guide
- [ ] Zillion: Setup Guide

**Comment by ShinyNT** [comment]
unfortunately I just play a minority of all games supported for AP so I really can't say I'll be willing to help with a lot of them, but I'll eventually do a few ones more I had planned to do already; like SMW, DKC3, maybe Terraria (when I decide to finally play it on AP) and the upcoming (hopefully for the next AP version) Yoshi's Island and Pokemon Emerald
I have set clique before too, I can probably do it for that game too (I mean it's clique after all)

**Comment by BootsinSoots** [comment]
Anything getting translated is a big help. Thank you for your support!

**Comment by nachocua** [comment]
Can I help with the translation of the Pokémon Emerald and Landstalker setup pages?

**Comment by BootsinSoots** [comment]
Of course! The discord is open if you have questions, but we can answer any you have here as well

**Comment by nachocua** [comment]
I understand that I must make a new branch and then make a push request for the merge to be done, right?

**Comment by BootsinSoots** [comment]
Yes
